### ISSUE TITLE
About Pose Results

### ISSUE THREAD

**Issue by cashiwamochi** [ClementPinard/SfmLearner-Pytorch#42]
Thanks for your cool work. This implementation is very useful. 
I have a question about the accuracy of poses in Seq 09, 10.
`./data/test_scenes.txt` doesn't have 09, 10.
According to KITTI-odometry-development-kit, 
```
Seq09 -> 2011_09_30_drive_0033 000000 001590       
Seq10 -> 2011_09_30_drive_0034 000000 001200
```
This means that this frameworks uses these 2 sequences in training or validation.
( Is this right ? )
How did you get the pose results on README ? 
Did you use 09, 10 in training, or did you use another method ?
Also I generated kitti-odometry-data using [this script](https://github.com/tinghuiz/SfMLearner/blob/master/data/prepare_train_data.py), and then tried using your framework. (with `--dataset-format stacked`). 
However, photometric-error in validation didn't decrease.
This is an example command.
```shell
python train.py /hoge/kitti_odom/ -b4 -m0.3 -s1.8 --epoch-size 1000 --sequence-length 5 --log-output  --rotation-mode euler --dataset-format stacked
```
The original paper uses 00-08 in training. I want to reproduce that score, but I cannot.
( Later, I will add Seq09,10 to `test_scenes.txt` and start training. )
Can you tell me your method ? 
Thank you for reading.

**Comment by ClementPinard** [comment]
Hello,

yes you are right ! the data separation  between train, validation and test is supposed to follow Eigen's split, which means that, as you said, sequences 9 and 10 are either in training or in valid.

Unfortunately, not much more is done to avoid overfitting on these scenes, as the split between train and test is random (whether it is from this repo or [Zhou's repo](https://github.com/tinghuiz/SfMLearner/blob/master/data/prepare_train_data.py#L90)). It could be nice to explicitely dismiss sequences 9 and 10 from being in the training set.

It appears that sequences 9 and 10 are indeed considered test sequences in the odom layer [here](https://github.com/tinghuiz/SfMLearner/blob/master/data/kitti/kitti_odom_loader.py#L21), but I am not sure what was the network used when testing pose, was the network trained on Odom (0-8) when testing pose and trained on Eigen split when testing depth ?

The pose results on README where obtained with a training on a random split so it might have overfitting.

I am aware that this test is not very thorough, because my personal goal was to obtain good results on depth.

**Comment by cashiwamochi** [comment]
Thanks!
As you said, I got good results on depth.
I will try what I wrote.
